src/systems/ec/implementations/react/components/icon/src/Icon.jsx

```diff
@@ -37,7 +37,7 @@ const Icon = ({
     >
       {title && <title {...(titleId && { id: titleId })}>{title}</title>}
       {desc && <desc {...(descId && { id: descId })}>{desc}</desc>}
-      <use xlinkHref={`${iconPath}#${shape}`} />
+      <use xlinkHref={`${iconPath}#${shape}`} aria-hidden="true" />
```

Comment:
there is already an aria-hidden attribute on the parent svg tag, so this one is not needed